ws-nextjs-app/pages/[service]/live/[id]/[[...variant]].page.tsx

```diff
@@ -133,7 +135,12 @@ export const getServerSideProps: GetServerSideProps = async context => {
     rendererEnv: 'test', // TODO: remove hardcoding
   });
 
-  logger.info(ROUTING_INFORMATION, {
+  let routingInfoLogger = logger.debug;
+  if (data.status !== OK) {
+    routingInfoLogger = logger.error;
+  }
```

Comment:
@amoore108 and @shayneahchoon are you OK with this logic - based on comments from @andrewscfc on this PR - we think that routing information should be set to `debug` rather than `info`, but if that if there was a non-200 status, then we would log this as an error so that we can still see the logs in Cloudwatch

Reply:
Yea all good from me 👍 We can test it out with the Live pages anyway since they are Test only atm.

Reply:
On test, the logging level is debug, so you will still see the logs in Cloudwatch on the test environment. Maybe worth making a note of checking the logs once the page is live?